programs/launcher/main.cpp

```diff
@@ -372,12 +372,12 @@ struct launcher_def {
   void format_ssh (const string &cmd, const string &host_name, string &ssh_cmd_line);
   bool do_ssh (const string &cmd, const string &host_name);
   void prep_remote_config_dir (eosd_def &node, host_def *host);
-  void launch (eosd_def &node, string &gts);
+  void launch (const string &eosdBinary, eosd_def &node, string &gts);
```

Comment:
stylistically we prefer under_scores to camelCase

Reply:
fixed